### PR TITLE
Add option fetchUser

### PR DESCRIPTION
**Pull request by ernestas-poskus**
Adding option fetchUser which allows control of fetching authenticated user.

cc @pi0

### CODE REVIEW

File: lib/templates/auth.store.js [hunk 1]
```diff
@@ -146,7 +146,9 @@ export default {
       <% } %>
 
       // Fetch authenticated user
+      <% if (options.user.fetchUser) { %>
```
Comment (pi0) [inline lib/templates/auth.store.js]: Thanks for this PR. What's your idea if we conditionally remove fetchUser action from store too when this option is `false`?

Comment (ernestas-poskus) [reply lib/templates/auth.store.js]: In my project I don't keep a state of user, token is enough therefore I wanted to make this conditional

**Comment by pi0** [comment]
@ernestas-poskus I think we need to update this lines too : 

- https://github.com/nuxt-community/auth-module/blob/master/lib/templates/auth.store.js#L16 (Only checking user when fetchUser is true)
- https://github.com/nuxt-community/auth-module/blob/master/lib/templates/auth.store.js#L16 (Remove extra variable here when fetchUser is false)

**Comment by ernestas-poskus** [comment]
```
https://github.com/nuxt-community/auth-module/blob/master/lib/templates/auth.store.js#L16 (Only checking user when fetchUser is true)
https://github.com/nuxt-community/auth-module/blob/master/lib/templates/auth.store.js#L11 (Remove extra variable here when fetchUser is false)
```

maybe better name would be `user.enabled` to be consistent with `token.enabled`?

**Comment by ernestas-poskus** [comment]
updated and renamed to `.user.enabled`

**Comment by ernestas-poskus** [comment]
offtopic: is there a way to handle API error after dispatching?

```
store.dispatch('auth/login', {
  fields: {
    username: 'your_username',
    password: 'your_password'
  }
})
```

**Comment by pi0** [comment]
Thanks for your contribution. Now changes seems good enough for merging...

For dispatch error handling can't ce just await on store.dispatch like this ?

```js
try {
await store.dispatch('auth/login', {
  fields: {
    username: 'your_username',
    password: 'your_password'
  }
})
} catch (e) {
 // Handle error
}
```

**Comment by ernestas-poskus** [comment]
I will try this, thanks !

**Pull request merged**